src/main/java/ca/corefacility/bioinformatics/irida/ria/web/ajax/CartAjaxController.java

```diff
@@ -51,9 +54,9 @@ public CartAjaxController(UICartService service) {
 	 * @param request Request to move a sample from the cart
 	 * @return the number of samples currently in the cart
 	 */
-	@DeleteMapping("/sample")
-	public ResponseEntity<Integer> removeSample(@RequestBody RemoveSampleRequest request) {
-		return ResponseEntity.ok(service.removeSample(request));
+	@DeleteMapping("/sample/{sampleId}")
+	public ResponseEntity<Integer> removeSample(@PathVariable Long sampleId) {
+		return ResponseEntity.ok(service.removeSample(sampleId));
```

Comment:
Add a response to the UI to display a notification that a sample was removed and also if a project is removed from the cart